phpunit.xml.dist

```diff
@@ -28,4 +27,5 @@
             <directory suffix=".php">src</directory>
         </whitelist>
     </filter>
+
```

Comment:
I had added the env config to that file as well, but later I've removed it, that's why there a leftover space there. I'll remove it 👍